lib/src/main/java/org/esbtools/lightbluenotificationhook/NotificationHook.java

```diff
@@ -262,14 +262,25 @@ private NotificationEntity makeNotificationEntityWithIncludedFields(HookDoc hook
         return notificationEntity;
     }
 
-    private void flatten(String prefix,JsonNode node,
-                         List<NotificationEntity.PathAndValue> removedPaths) {
-        JsonNodeCursor cursor=new JsonNodeCursor(Path.EMPTY,node);
+    private void flatten(String prefix, JsonNode node, List<PathAndValue> entityData) {
```

Comment:
Maybe it's because I've been looking at functional code lately, but I'd prefer if this just returned `List<PathAndValue>`, instead of modifying passed in `entityData`. My 2 cents.

No biggie though.